pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -160,12 +188,16 @@ func activeQComp(podInfo1, podInfo2 interface{}) bool {
 }
 
 // NewPriorityQueue creates a PriorityQueue object.
-func NewPriorityQueue(stop <-chan struct{}, fwk framework.Framework) *PriorityQueue {
-	return NewPriorityQueueWithClock(stop, util.RealClock{}, fwk)
-}
+func NewPriorityQueue(
```

Comment:
how can i configure the max duration if i do not want to default it ?

Reply:
This is a pattern which used in the kubernetes codebase.

```go
NewPrioirtyQueue(
  stopCh,
  fwk.
  internalqueue.WithPodMaxBackoffDuration(500*time.Second),
  internalqueue.WithClock(xx),
)
```

Reply:
I mean how can i  set the max duration value, any flag for that ? or add it somewhere ?

Reply:
Every time i want to change the value, i need to modify the code ?